README.md

```diff
@@ -167,4 +169,16 @@ using your real name (sorry, no pseudonyms or anonymous contributions.)
 
 You can add the sign off when creating the git commit via `git commit -s`.
 
+### Commit Style
+
+Simple house-keeping for clean git history.
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
```

Comment:
On Wed, Sep 09, 2015 at 04:12:24PM -0700, Vincent Batts wrote:
> > +2. Limit the subject line to 50 characters
> 
> this is actually from `git-commit(1)` own guidelines.

Then I think everyone is just blindly copying it around ;).  Looking
at the git.git history through v2.3.4 (git log --no-merges --format=%s
v2.3.4), we have 29853 commits, with 56% ≤ 50 chars and 94% ≤ 70
chars.